rules/models/changes/mapping.go

```diff
@@ -0,0 +1,60 @@
+package changes
+
+import "github.com/m3db/m3metrics/rules/models"
+
+// MappingRuleChange is a mapping rule diff.
+type MappingRuleChange struct {
+	Op       UpdateOp            `json:"updateOp"`
```

Comment:
As mentioned below, might be more succinct to just call the type `Op` and use `json:"op"` instead.

Reply:
O yeah. I think this may have been a mistake.